src/search-scenario/azext_search_scenario/custom.py

```diff
@@ -0,0 +1,312 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import telemetry
+from azure.cli.core.style import Style, print_styled_text
+
+from .constants import MatchRule, SearchScope
+from .requests import get_search_result_from_api
+from .utils import get_int_option
+
+
+def search_scenario(cmd, search_keyword, scope=None, match_rule=None, top=None):
```

Comment:
Is it better to use `search` as the name of the function as it not only searches scenario, but also commands?

Reply:
Done